curriculum/challenges/_meta/learn-how-to-talk-about-hobbies-and-interests/meta.json

```diff
@@ -10,8 +10,444 @@
+    },
+    {
+      "id": "657fba163fec41e8e00c5817",
+      "title": "Task 71"
+    },
+    {
+      "id": "657fbad88163d8e96189f823",
+      "title": "Task 72"
+    },
+    {
+      "id": "657fbb3a1a63d0e9c307fd83",
+      "title": "Task 73"
+    },
+    {
+      "id": "657fbc14d41a20ea5f286378",
+      "title": "Task 74"
+    },
+    {
+      "id": "657fbc85c25a16eac8356182",
+      "title": "Task 75"
+    },
+    {
+      "id": "657fbcecfc4d42eb28700349",
+      "title": "Task 76"
+    },
+    {
+      "id": "657fbd591461eaeba07ffff1",
+      "title": "Task 77"
+    },
+    {
+      "id": "657fbde9a43e35ec1ebafe56",
+      "title": "Task 78"
+    },
+    {
+      "id": "657fff0bfb6a28f1d70fa9ef",
+      "title": "Task 79"
+    },
+    {
+      "id": "657fff7dabba2ff23993b08c",
+      "title": "Task 80"
+    },
+    {
+      "id": "6580001adc7fd4f2b244f3a5",
+      "title": "Task 81"
+    },
+    {
+      "id": "65800082405352f30c6dbc7a",
+      "title": "Task 82"
+    },
+    {
+      "id": "65800148406738f397561d77",
+      "title": "Task 83"
+    },
+    {
+      "id": "658001a018bbcbf3fd84f832",
+      "title": "Task 84"
+    },
+    {
+      "id": "6580026241ae0ef46b181e49",
+      "title": "Task 85"
+    },
+    {
+      "id": "658002d23e245ff4ca8542d5",
+      "title": "Task 86"
+    },
+    {
+      "id": "658003870281a9f5541085af",
+      "title": "Task 87"
+    },
+    {
+      "id": "658003d8ff9da6f5c08971a1",
+      "title": "Task 88"
+    },
+    {
+      "id": "658009d86dc9caf988e2ea64",
+      "title": "Task 89"
+    },
+    {
+      "id": "65800a95389cc0fa4c197587",
+      "title": "Task 90"
+    },
+    {
+      "id": "65800b1c13005dfa9df75d73",
+      "title": "Task 91"
+    },
+    {
+      "id": "65800b96989013fb24aa1b70",
+      "title": "Task 92"
+    },
+    {
+      "id": "65800c12978ba7fb82007446",
+      "title": "Task 93"
+    },
+    {
+      "id": "65800cf36faba0fbfa1027b6",
+      "title": "Task 94"
+    },
+    {
+      "id": "65800d61890343fc5cce0ec8",
+      "title": "Task 95"
+    },
+    {
+      "id": "65800df7fc5d49fcd7209248",
+      "title": "Task 96"
+    },
+    {
+      "id": "65800e60dad5fefd4ed91589",
+      "title": "Task 97"
+    },
+    {
+      "id": "65800f4d194382fdebb81e1f",
+      "title": "Task 98"
+    },
+    {
+      "id": "658010478daa16fe79d8113a",
+      "title": "Task 99"
+    },
+    {
+      "id": "65801182280f63ff10ca4d4f",
+      "title": "Task 100"
+    },
+    {
+      "id": "658011ef9ec114ff80ce5e42",
+      "title": "Task 101"
+    },
+    {
+      "id": "658013bd3b1a06001a59e006",
+      "title": "Task 102"
+    },
+    {
+      "id": "65802ee9706eb103aea442f8",
+      "title": "Task 103"
+    },
+    {
+      "id": "65802f717cef8c042af950b8",
+      "title": "Task 104"
+    },
+    {
+      "id": "65802fe92ef0f404ba0437f7",
+      "title": "Task 105"
+    },
+    {
+      "id": "658030876ac4f605145aeae1",
+      "title": "Dialogue 5: Getting into Extreme Sports"
```

Comment:
```suggestion
      "id": "657ec6ac4de4eac8bab2f2a7",
      "title": "Task 39"
    },
    {
      "id": "657ef0061e99a3ddd0e1245f",
      "title": "Dialogue 3: The Collector"
    },
    {
      "id": "657eda6e48e0d7c92f0af163",
      "title": "Task 40"
    },
    {
      "id": "657edbc3e12e35cfc1d80358",
      "title": "Task 41"
    },
    {
      "id": "657edc5136791ed04ffab234",
      "title": "Task 42"
    },
    {
      "id": "657ee0a0c52d0ed1164a85d8",
      "title": "Task 43"
    },
    {
      "id": "657ee214b9ad1ad2b6f8325c",
      "title": "Task 44"
    },
    {
      "id": "657ee25acee11cd3122e0876",
      "title": "Task 45"
    },
    {
      "id": "657ee2f44346f3d4017cc990",
      "title": "Task 46"
    },
    {
      "id": "657ee3954d64e5d465500620",
      "title": "Task 47"
    },
    {
      "id": "657ee4171371e9d4d1402e91",
      "title": "Task 48"
    },
    {
      "id": "657ee47c1af836d5289254d9",
      "title": "Task 49"
    },
    {
      "id": "657ee51ce8dac8d5a227f758",
      "title": "Task 50"
    },
    {
      "id": "657ee5ddb47b1dd62f87d784",
      "title": "Task 51"
    },
    {
      "id": "657ee66dd1216ad6c61f1168",
      "title": "Task 52"
    },
    {
      "id": "657ee781258676d764a5282f",
      "title": "Task 53"
    },
    {
      "id": "657ee8a9a195b8d8756ca5e1",
      "title": "Task 54"
    },
    {
      "id": "657ee957cb4719d9031a0be4",
      "title": "Task 55"
    },
    {
      "id": "657ee9b69c2e26d964f67ee4",
      "title": "Task 56"
    },
    {
      "id": "657eeb163e1993d9e342d661",
      "title": "Task 57"
    },
    {
      "id": "657eebaa042b5cda6ec2dac9",
      "title": "Task 58"
    },
    {
      "id": "657eec0a21bc7adada04453e",
      "title": "Task 59"
    },
    {
      "id": "657eecb3b368badb3cc7fe4c",
      "title": "Task 60"
    },
    {
      "id": "657eed5ed318e4dbbce6903a",
      "title": "Task 61"
    },
    {
      "id": "657ef4e5a4a1e1e126eba6dd",
      "title": "Dialogue 4: Inviting to A Convention"
    },
    {
      "id": "657ef58855ff45e1e4ca70ee",
      "title": "Task 62"
    },
    {
      "id": "657ef61be0a682e2591776f6",
      "title": "Task 63"
    },
    {
      "id": "657fb48f31654ae4cb52d395",
      "title": "Task 64"
    },
    {
      "id": "657fb5034da144e55cbee9d7",
      "title": "Task 65"
    },
    {
      "id": "657fb5afeeba2de5d01dda0e",
      "title": "Task 66"
    },
    {
      "id": "657fb665a39478e642f5a139",
      "title": "Task 67"
    },
    {
      "id": "657fb8cfbd0e4ae797fc6077",
      "title": "Task 68"
    },
    {
      "id": "657fb92c6f888fe8013f1a28",
      "title": "Task 69"
    },
    {
      "id": "657fb980a9b567e860b77f2c",
      "title": "Task 70"
    },
    {
      "id": "657fba163fec41e8e00c5817",
      "title": "Task 71"
    },
    {
      "id": "657fbad88163d8e96189f823",
      "title": "Task 72"
    },
    {
      "id": "657fbb3a1a63d0e9c307fd83",
      "title": "Task 73"
    },
    {
      "id": "657fbc14d41a20ea5f286378",
      "title": "Task 74"
    },
    {
      "id": "657fbc85c25a16eac8356182",
      "title": "Task 75"
    },
    {
      "id": "657fbcecfc4d42eb28700349",
      "title": "Task 76"
    },
    {
      "id": "657fbd591461eaeba07ffff1",
      "title": "Task 77"
    },
    {
      "id": "657fbde9a43e35ec1ebafe56",
      "title": "Task 78"
    },
    {
      "id": "657fff0bfb6a28f1d70fa9ef",
      "title": "Task 79"
    },
    {
      "id": "657fff7dabba2ff23993b08c",
      "title": "Task 80"
    },
    {
      "id": "6580001adc7fd4f2b244f3a5",
      "title": "Task 81"
    },
    {
      "id": "65800082405352f30c6dbc7a",
      "title": "Task 82"
    },
    {
      "id": "65800148406738f397561d77",
      "title": "Task 83"
    },
    {
      "id": "658001a018bbcbf3fd84f832",
      "title": "Task 84"
    },
    {
      "id": "6580026241ae0ef46b181e49",
      "title": "Task 85"
    },
    {
      "id": "658002d23e245ff4ca8542d5",
      "title": "Task 86"
    },
    {
      "id": "658003870281a9f5541085af",
      "title": "Task 87"
    },
    {
      "id": "658003d8ff9da6f5c08971a1",
      "title": "Task 88"
    },
     {
      "id": "658030876ac4f605145aeae1",
      "title": "Dialogue 5: Getting into Extreme Sports"
      },
    {
      "id": "658009d86dc9caf988e2ea64",
      "title": "Task 89"
    },
    {
      "id": "65800a95389cc0fa4c197587",
      "title": "Task 90"
    },
    {
      "id": "65800b1c13005dfa9df75d73",
      "title": "Task 91"
    },
    {
      "id": "65800b96989013fb24aa1b70",
      "title": "Task 92"
    },
    {
      "id": "65800c12978ba7fb82007446",
      "title": "Task 93"
    },
    {
      "id": "65800cf36faba0fbfa1027b6",
      "title": "Task 94"
    },
    {
      "id": "65800d61890343fc5cce0ec8",
      "title": "Task 95"
    },
    {
      "id": "65800df7fc5d49fcd7209248",
      "title": "Task 96"
    },
    {
      "id": "65800e60dad5fefd4ed91589",
      "title": "Task 97"
    },
    {
      "id": "65800f4d194382fdebb81e1f",
      "title": "Task 98"
    },
    {
      "id": "658010478daa16fe79d8113a",
      "title": "Task 99"
    },
    {
      "id": "65801182280f63ff10ca4d4f",
      "title": "Task 100"
    },
    {
      "id": "658011ef9ec114ff80ce5e42",
      "title": "Task 101"
    },
    {
      "id": "658013bd3b1a06001a59e006",
      "title": "Task 102"
    },
    {
      "id": "65802ee9706eb103aea442f8",
      "title": "Task 103"
    },
    {
      "id": "65802f717cef8c042af950b8",
      "title": "Task 104"
    },
    {
      "id": "65802fe92ef0f404ba0437f7",
      "title": "Task 105"
```

Tasks for dialogue 3 and 5 were out of place. I've made this change so they appear in the correct order.